plugins/readme/plugin_tests/readmeSpec.js

```diff
@@ -0,0 +1,51 @@
+girderTest.importPlugin('readme');
```

Comment:
Does this spec actually get run? Is it harnessed by CTest or pytest?

Reply:
AFAICT all the plugin frontend tests are run through CTest. It's run through CTest. You can see the logs where it ran in CI:
```
test 76
      Start 76: web_client_readme.readme

76: Test command: /home/circleci/project/girder_env/bin/python "-m" "unittest" "-v" "tests.web_client_test"
...
76: Loaded plugin "readme"
...
76: . Test the readme UI -> register a user
76: /home/circleci/project/girder/girder/models/user.py:613: DeprecationWarning: count is deprecated. Use Collection.count_documents instead.
76:   return folders.count()
76: /home/circleci/project/girder/girder/api/rest.py:441: DeprecationWarning: count is deprecated. Use Collection.count_documents instead.
76:   cherrypy.response.headers['Girder-Total-Count'] = val.count()
78: . Create and log in to a user for testing -> logout
76: /home/circleci/project/girder/girder/models/folder.py:602: DeprecationWarning: count is deprecated. Use Collection.count_documents instead.
76:   return self.childItems(folder, fields=()).count()
76: /home/circleci/project/girder/girder/models/folder.py:623: DeprecationWarning: count is deprecated. Use Collection.count_documents instead.
76:   return folders.count()
76: . Test the readme UI -> navigate to the user's Public folder
76: . Test the readme UI -> does not render the README when there is no README
76: . Test the readme UI -> upload README.md
76: . Test the readme UI -> renders the README
76: 
76: Test the readme UI: 8 of 8 passed.
76: ----------------
76: Testing Finished
76: ----------------
76: 5 specs, 0 failures in 1.334s.
76: 
...
76: ok
76: 
76: ----------------------------------------------------------------------
76: Ran 1 test in 4.801s
76: 
76: OK
```

Reply:
Thanks. I forgot that gets pulled in by the standard plugin tests thing.